operators/src/source/gdal_source.rs

```diff
@@ -96,13 +98,26 @@ pub enum GdalLoadingInfoPartIterator {
     Dynamic {
         time_step_iter: TimeStepIter,
         params: GdalDatasetParameters,
-        placeholder: String,
-        time_format: String,
+        time_placeholders: HashMap<String, GdalSourceTimePlaceholder>,
```

Comment:
maybe name it `file_time_injection`? idk but it's more than a placeholder.

Also: do we have to check that there is at least one placeholder defined?

Reply:
The name is more confusing to the user as *placeholder* I guess.

We don't need to check that one is defined. If there is none, there is just no replacement, right?

Reply:
> The name is more confusing to the user as placeholder I guess.

ok

> We don't need to check that one is defined. If there is none, there is just no replacement, right?

if no placeholder is defined, the time is irrelevant and the user should use static meta data instead.

Reply:
So we should fail on deserialization?
But what if the user specifies one but doesn't use it in the file_path?

Reply:
if we want to enforce this we should either fail on deserialization or on operator initialization.

pro:
 - fail early, otherwise it might be hard to figure out why the data is not like desired

contra:
- inconvienent for the user. maybe they have only one file currently and don't know the pattern but want to define the dataset as timer series already

Reply:
3345c04